score/etl/datasets/ejscreen_2020.py

```diff
@@ -0,0 +1 @@
+# https://gaftp.epa.gov/EJSCREEN/2020/EJSCREEN_2020_StatePctile.csv.zip
```

Comment:
What is the point of this file? If it's intentional to keep it around, can we add some comments explaining it?

Reply:
That is where the final python code will live once the notebook is finalized.

Reply:
Ok - can we add that comment to the codebase? Thanks!